### PR TITLE
build video instead of gif

### PR DESCRIPTION
**Pull request by wdwatkins**
This works, but a few things need to be hammered out yet:

-  Need a windows person to try this — it assumes you can run `ffmpeg` in your terminal, so the executable needs to go somewhere on your system path variable.  Download it here: https://www.ffmpeg.org/download.html

- I was struggling with getting ffmpeg to accept an explicit list of input files, rather than just a pattern.  I think it is possible.  Right now though, it is just creating the gif from every file matching this pattern: `6_visualize/tmp/frame_*.png`.  So if there were for some reason images from a previous run that were no longer being used and were still in that directory, they would get lumped into the video.  I'm sure we can account for that if needed though.

### CODE REVIEW

**Comment by lindsaycarr** [comment]
I had to follow [these instructions](http://blog.gregzaal.com/how-to-install-ffmpeg-on-windows/) to download and install `ffmpeg` on windows. The files to download at [this URL](https://www.ffmpeg.org/download.html) couldn't be opened by my computer.

I think I'm having the opposite problem to your second bullet. When I run `scipiper::scmake('6_visualize/out/year_in_review.webm', remake_file = '6_visualize.yml', force = TRUE)`, when `task_names = NULL` until that task in `6_visualize.yml` it throws this error:

```
'6_visualize/tmp/frame_*.png': No such file or directory
Error in target_run(target, store, quiet) : 
  command for 6_visualize/out/year_in_review.webm did not create file
```

**Comment by wdwatkins** [comment]
You've confirmed that there actually are files that match `6_visualize/tmp/frame_*.png`?

If there are, can you try the command directly from the terminal? 
`ffmpeg -framerate 1 -pattern_type glob -i '6_visualize/tmp/frame_*.png'`

**Comment by wdwatkins** [comment]
Ok @lindsaycarr hopefully this is windows-proof now.

File: 6_visualize.yml [hunk 1]
```diff
@@ -150,11 +150,11 @@ targets:
...
+
+  6_visualize/out/year_in_review.webm:
     command: combine_animation_frames(
       gif_file=target_name,
-      animation_cfg=animation_cfg,
-      task_names=NULL)
-  6_visualize/out/year_in_review.gif.ind:
-    command: gd_put(target_name, '6_visualize/out/year_in_review.gif')
+      animation_cfg=animation_cfg)
+  6_visualize/out/year_in_review.web.ind:
```
Comment (lindsaycarr) [inline 6_visualize.yml]: does this need to me webm?

**Comment by lindsaycarr** [comment]
Fixes #47

File: 6_visualize.yml (hunk 1 above)
Comment (wdwatkins) [reply 6_visualize.yml]: yes

**Pull request merged**